ddtrace/contrib/psycopg/patch.py

```diff
@@ -142,18 +142,10 @@ def _unpatch(psycopg_module):
             _u(psycopg_module, "Cursor")
             _u(psycopg_module, "AsyncCursor")
 
-            try:
-                _u(psycopg_module.Connection, "connect")
-                _u(psycopg_module.AsyncConnection, "connect")
-
-            # _u throws an attribute error for Python 3.11 on method objects because of
-            # no __get__ method on the BoundFunctionWrapper
-            except AttributeError:
-                _original_connection_class = config.psycopg["_patched_functions"]["psycopg.Connection"]
-                _original_asyncconnection_class = config.psycopg["_patched_functions"]["psycopg.AsyncConnection"]
-
-                psycopg_module.Connection = _original_connection_class
-                psycopg_module.AsyncConnection = _original_asyncconnection_class
+            # _u throws an attribute error for Python 3.11, no __get__ on the BoundFunctionWrapper
```

Comment:
Can we look into making the BoundFunctionWrapper compatible with python3.11? I can see this issue coming up in other integrations.

Reply:
will look into this